src/containers/ProvisionerForm.js

```diff
@@ -8,6 +8,8 @@ import GithubService from '../components/provisionerForm/GithubService';
 import ProjectName from '../components/provisionerForm/ProjectName';
```

Comment:
I can see that this line is to import the js, what I'm asking is if there is another way to import the ProjectName (with an absolute path maybe?) (Because if this code is run in a different PATH, the import won't work)